tests/integration/llm/client.py

```diff
@@ -128,6 +134,14 @@ def batch_generation(batch_size):
         input_sentences *= math.ceil(batch_size / len(input_sentences))
     return input_sentences[:batch_size]
 
+def t5_batch_generation(batch_size):
+    input_sentences = [
+        "translate English to German: The house is wonderful.",
+        "translate English to German: My name is AWS",
```

Comment:
Could we also add some other task other than translation?

Reply:
yes, plan to add more tests. Want to test the entire workflow in this PR first.

Reply:
replaced one case with summarization.